build/helpers/coverage.js

```diff
@@ -281,13 +282,7 @@ function parseFunctionCoverage(script, map, range, name) {
  * @return {BlockCoverage | null}
  */
 function parseBlockCoverage(script, map, range) {
-  let parsed = parseRange(script, map, range, {
-    trim: true
-  });
-
-  if (parsed === null) {
-    parsed = parseRange(script, map, range);
-  }
+  let parsed = parseRange(script, map, range);
```

Comment:
The non-trimming retry is needed in cases where trimming causes source maps to refuse mapping the given range.